File: 1630-count-odd-numbers-in-an-interval-range/count-odd-numbers-in-an-interval-range.go
func countOdds(low int, high int) int {
    oddCountTillHigh := (high + 1) / 2;
    oddCountTillLow := (low + 1) / 2;
    oddCount := oddCountTillHigh - oddCountTillLow;
    oddCount += low % 2;
    return oddCount;
}

File: 1630-count-odd-numbers-in-an-interval-range/count-odd-numbers-in-an-interval-range.js
/**
 * @param {number} low
 * @param {number} high
 * @return {number}
 */
var countOdds = function(low, high) {
    var oddCountTillHigh = Math.floor((high + 1) / 2);
    var oddCountTillLow = Math.floor((low + 1) / 2);
    var oddCount = oddCountTillHigh - oddCountTillLow;
    oddCount += (low % 2);
    return oddCount;
};